core/src/com/unciv/models/translations/TranslationFileWriter.kt

```diff
@@ -344,6 +344,8 @@ object TranslationFileWriter {
                         UniqueParameterType.guessTypeForTranslationWriter(parameter, ruleset).parameterName
                     }
                 parameterNames.addNumberedParameter(parameterName)
+                if (parameterName == UniqueParameterType.Unknown.parameterName)
```

Comment:
Shouldn't it (also) be `UniqueParameterType.Comment`?

Reply:
No, the guessing order doesn't consider that one. Predetermined guessing order (in UniqueParameterType.Companion) was an evil I didn't like but after testing seemed necessary.